### ISSUE TITLE
simplify directory structure

### ISSUE THREAD

**Issue by thijsheus** [microhh/microhh#182]
- [ ] scaling and scheme may no longer be necessary, or move to misc
- [ ] old_cases may be able to go, or move to cases
- [ ] captures and wisdom may be able to move to tuner
- [ ] rte-rrtmgp should go to external

**Comment by Chiil** [comment]
Agree, let's do this once the main is fully operational again. I think that having all submodules in `external` is good. I would like to retain the scripts in `scheme` somewhere, we can move it into `misc` but I would keep it in its directory. The `scaling` directory contains the numbers that are in the GMD paper, we could remove them or move it also into `misc` if we want to retain it.

**Comment by thijsheus** [comment]
Removing this from the 2.0 milestone if we want to do it after the tag
